.circleci/config.yml

```diff
@@ -744,6 +744,16 @@ jobs:
         no_output_timeout: "1h"
         command: |
             source "/pytorch/.circleci/scripts/binary_linux_build.sh"
+    - run:
+        name: save binary size
+        no_output_timeout: "5m"
+        command: |
+            source /env
+            cd /pytorch && export COMMIT_TIME=$(git log --max-count=1 --format=%ct || echo 0)
+            pip3 install requests && \
+            SCRIBE_GRAPHQL_ACCESS_TOKEN=${SCRIBE_GRAPHQL_ACCESS_TOKEN} \
```

Comment:
Hmm... is this cred living in a secure context? I didn't see anywhere you attached the context (and this job is the build job, so it shouldn't).

Reply:
it's an environment variable defined in pytorch setting's page, like other AWS related variables